### PR TITLE
added state diagrams

### PR DESCRIPTION
**Pull request by John-Merritt**
image

image

reservation use case is the same just with random accidental text removed and authentication added image

### CODE REVIEW

**Review by mclhtay — DISMISSED** [review]
A couple of things have changed with the newest commits:

Employee:
1) Start menu state can go to terminate state by selecting quit
2) Admin menu can go back to the start menu state by selecting back

User:
1) Start menu state can go to terminate state by selecting quit
2) After registering, the user is taken to the starting menu state instead of log-in
3) User Menu can return to start menu state by selecting back

**Review by mclhtay — COMMENTED** [review]
LGTM but since this is a system-wide diagram, let's get another approval before merging it in.

**Review by vzhang23 — APPROVED** [review]
lgtm

**Pull request merged**
